nsqd/http.go

```diff
@@ -596,3 +600,41 @@ func (s *httpServer) printStats(stats []TopicStats, health string, startTime tim
 	}
 	return buf.Bytes()
 }
+
+type allowedOpts struct {
+	NSQLookupdTCPAddresses []string `json:"nsqlookupd_tcp_addresses"`
```

Comment:
After a discussion offline, what if we just implemented `PUT /config/<option>` for now, which leaves room for some of these future decisions (the body is the JSON encoded value).

The `<option>` would match the equivalent key in the config file?